src/main/java/com/hbm/inventory/recipes/ElectrolyserFluidRecipes.java

```diff
@@ -24,9 +26,10 @@ public class ElectrolyserFluidRecipes extends SerializableRecipe {
 
 	@Override
 	public void registerDefaults() {
-		recipes.put(Fluids.WATER, new ElectrolysisRecipe(2_000, new FluidStack(Fluids.HYDROGEN, 200), new FluidStack(Fluids.OXYGEN, 200)));
-		recipes.put(Fluids.HEAVYWATER, new ElectrolysisRecipe(2_000, new FluidStack(Fluids.DEUTERIUM, 200), new FluidStack(Fluids.OXYGEN, 200)));
+		recipes.put(Fluids.WATER, new ElectrolysisRecipe(2_000, new FluidStack(Fluids.HYDROGEN, 200), new FluidStack(Fluids.OXYGEN, 200),10));
```

Comment:
we had a more than lengthy argument about this when the recipes were rebalanced, i don't see half process time as justified

Reply:
I tested it extensively, and this allows the electrolyzer to roughly catch up to the chemfac, and be better than it at electrolysis for the cost